wazuh/wazuh-dashboard-plugins
# The check update button does not show the date of the last scan

### Description

Fixed the date of the last update scan in the tooltip of the check updates button.

Fixed the menu overlay with update notification when opening and closing a flyout.
 
### Issues Resolved

- #6308 

### Evidence

![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/63758389/932eded8-0057-4b14-ad5b-ba6526210bc7)


https://github.com/wazuh/wazuh-dashboard-plugins/assets/63758389/6813bda4-3b75-4887-929f-49284c056848

### Test

1. Navigate to 'Server APIs'
2. Check the date in the check updates tooltip and if you click it should change.


You must have the update notification available.
1. Navigate to 'Security'
2. Click on 'Create user'
3. Close flyout
4. The notification should look complete

### Check List
